xAPI.md

```diff
@@ -2488,6 +2487,15 @@ definitions for multiple identifiers. E.g. http://example.com/xapi/verbs#defenes
 
 ##### LRS Requirements
 * The LRS MAY act as a [Metadata Consumer](#def-metadata-consumer) and attempt to resolve identifier IRIs.
+* If an Activity IRI is an IRL, an LRS SHOULD attempt to GET that IRL, and include in HTTP
```

Comment:
In reticent to change this in this PR since this was just supposed to add back in stuff that got dropped, however for the sake of speed I'll sneak these in. If it gets controversial I'll change it back and it can be handled separately.